docs/detections/rules-ui-manage.asciidoc

```diff
@@ -104,6 +105,22 @@ NOTE: When duplicating a rule with exceptions, you can choose to duplicate the r
 * To enable or disable a single rule, switch on the rule's *Enabled* toggle.
 * To <<snooze-rule-actions,snooze>> actions for rules, click the bell icon.
 
+[float]
+[[manually-run-rules]]
+=== Manually run rules
+
+Manually run enabled rules during a specfied time range. 
```

Comment:
What are some use cases for running a rule manually? I remember talking about this in earlier UX design sessions, but don't recall specific uses. The instructions clearly explain _how_ to do manual runs, but some extra context could help users understand _why_ they'd want to do this.

Reply:
My notes from our UX design meetings show that this feature is useful for manually testing rules (as opposed to waiting for them to start running on their automated schedule) or for providing coverage during past gaps. Does this sound right @nkhristinin? Is there anything missing? Should the docs highlight different use cases?

Reply:
Gaps  and manually testing rule sounds right!

Reply:
That sounds like what I remember too. Thanks for confirming, @nkhristinin!

Reply:
@nkhristinin and @approksiu let me know if this revision works:

```suggestion
Manually run rules to promptly test or execute them during a specified period of time. 
```

Reply:
>or execute them during a specified period of time. 

It's not really "or", it's always specified period of time

Reply:
Maybe phrase like this: "Manually run rules for a specified time for testing or additional alerts coverage purposes."

Reply:
@nkhristinin I'm not sure I understand the error that you're pointing out. The conjunction "or" joins the first part of the sentence (_manually run rules to promptly test or execute them_) while the second half (_during a specified period of time_) applies to both. If you were to write out the longer version of the sentence, it'd be:
You can manually run rules to promptly test them during a specified period of time, or manually run rules to execute them during a specified period of time. 

@approksiu thanks for the suggestion. What do you think about swapping "additional alerts coverage" with "additional rules coverage"? For example:

```suggestion
Manually run enabled rules for a specfied period of time for testing purposes or additional rule coverage. 
```

Reply:
Merging this for now because I need to move these changes to a separate ESS PR.